syncto/authentication.py

```diff
@@ -38,5 +40,18 @@ def build_sync_client(request):
 
     bid_assertion = authorization_header.split(" ", 1)[1]
     client_state = request.headers[CLIENT_STATE_HEADER]
-    sync_client = SyncClient(bid_assertion, client_state)
+    try:
+        sync_client = SyncClient(bid_assertion, client_state)
+    except HTTPError as e:
+        if e.response.status_code in (400, 401):
```

Comment:
Apparently we can have a 403 on exceeded quota: https://github.com/mozilla-services/server-syncstorage/blob/d961d1012e2c48921180934c369b136b83916a7d/syncstorage/views/decorators.py#L134